includes/render.php

```diff
@@ -28,7 +28,7 @@ class GEM_Form_Renderer {
 	 * @return string
 	 */
 	public function process( $form_id, $echo = false ) {
-		$form = GEM_Dispatcher::get_fields( (int) $form_id );
+		$form  = (object) apply_filters( 'test', GEM_Dispatcher::get_fields( (int) $form_id ) );
```

Comment:
@EvanHerman Pretty sure `test` is not the tag name you had in mind here

Reply:
Whoops, that should have been removed :D